crates/bevy_core_pipeline/src/dof/mod.rs

```diff
@@ -889,7 +893,7 @@ impl DepthOfFieldPipelines {
 /// `sampler2DShadow` and will cheerfully generate invalid GLSL that tries to
 /// perform non-percentage-closer-filtering with such a sampler. Therefore we
 /// disable depth of field entirely on WebGL 2.
-#[cfg(target_arch = "wasm32")]
+#[cfg(all(feature = "webgl", target_arch = "wasm32", not(feature = "webgpu")))]
 fn depth_textures_are_supported() -> bool {
```

Comment:
Why is this a function and not a constant?

Reply:
That was just how the original PR was made and nobody commented on it. I guess I can update it to use a const

Reply:
I updated it to use a const instead since it indeed doesn't need to be a function